libs/atqr/domain/src/lib/domain-spec/stories/judgeChallenge.disl

```diff
@@ -0,0 +1,31 @@
+domain ATQR
+
+context Challenges
+
+//função no Challenge
+
+💬 RequestJudgement? :challengeId
+  rules:
+    challenge.status = ChallengeStatus.Overdue
```

Comment:
Did you mean to assign or compare?

Reply:
compare

Reply:
Then you need `==` for it to be a comparison